src/components/settings-row/settings-row.stories.mdx

```diff
@@ -44,6 +44,19 @@ All `children` are rendered in the input column to the right of the header colum
   </Story>
 </Canvas>
 
+### with HeadingType
+
+The `headingType` prop changes the HTML heading element of the `title` string which is displayed within the `SettingsRow` component. Each `string` 
+represents a HTML Section Heading element e.g `h1` is equal to `<h1>`, allowing up to six sections of headings from most to least important.
+
+<Canvas>
+  <Story name="headingType">
+    <SettingsRow headingType="h2" description="Description" title="Title">
```

Comment:
and again I'd prefer to use a "lower" heading for this example - although not the one set as the default if you implement my suggestion to change that from `h1`